### ISSUE TITLE
[a11y]: Issue in Accordion component at 400%

### ISSUE THREAD

**Issue by Vinodbs** [carbon-design-system/carbon#16201]
### Package

@carbon/react

### Browser

Chrome, Firefox

### Operating System

MacOS

### Package version

v1.50.0

### React version

v17.0.2

### Automated testing tool and ruleset

Manual

### Description

When Zoomed to 400% - in the Accordion Component ,Horizontal scrollbars appears. When the browser zoom is used to scale content to 400%, the Accordion component should reflow- i.e., it should be presented in one column so that scrolling in more than one direction is not necessary.

### Reproduction/example

https://stackblitz.com/edit/github-yxkecz?file=src%2FApp.jsx

### Steps to reproduce

On zoom , Accordion component reflow ( be presented in one column so that scrolling in more than one direction is not necessary) is not happening

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
